fast/stages/02-networking-nva/README.md

```diff
@@ -0,0 +1,343 @@
+# Networking with Network Virtual Appliance
+
+This stage sets up the shared network infrastructure for the whole organization.
+It is an alternative to the [02-networking-nva stage](../02-networking-nva/README.md).
```

Comment:
It is an alternative to 02-networking (and again, I'd rename 02-networking to 02-networking-vpn).
Or I would go as far as even remove this phrase.